build-scripts/version.sh

```diff
@@ -7,7 +7,7 @@
 # Current version of BlueChi
 VERSION=0.8.0
 # Specify if build is a official release or a snapshot build
-IS_RELEASE=false
+IS_RELEASE="0"
```

Comment:
Can we stick with the boolean?

Reply:
Boolean in shell scripts are usually represented as integers